README.rst

```diff
@@ -78,6 +78,28 @@ Want to know if a word you're proposing exists in codespell already? It is possi
     echo "word" | codespell -
     echo "1stword,2ndword" | codespell -
 
+Using a config file
+-------------------
+
+Command line options can also be specific in a config file.
+
+When running ``codespell``, it will check in the current directory for a file
+named ``setup.cfg`` or ``.codespellrc`` (or a file specified via ``--config``),
```

Comment:
Why two options, what you want to do if you have both? I would just use .codespell, keep it simple.

Reply:
If possible I'd do whatever Flake8 and other tools in the ecosystem do in terms of hierarchy/combination. It lowers the adoption burden by following the same rules as other programs.

Reply:
Configparse parses them in order and each will override the next, if it has any keys in that section. This is similar to how flake8 and other tools work as far as I know.

* setup.cfg is first,
* .codespellrc will override anything (if it exists)
* --config will override anything (if it exists)

Reply:
Okay

Reply:
Ahh flake8 does allow `--config`. Okay +1 for this order/hierarchy. In theory it should "just" be a loop over these filenames in that order and loading them if they exist...

Reply:
Yes, it already works as you are asking for. Please read the source code and configparser documentation:

https://github.com/codespell-project/codespell/pull/1668/files#diff-c4984ff60bb6d58a3a2f9940d1a396f9R384

https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read